src/irmin.mli

```diff
@@ -1626,11 +1388,12 @@ module Private: sig
       type commit
       (** The type for commit values. *)
 
-      val create: t -> node:node -> parents:commit list -> task:task -> commit Lwt.t
+      val v: t -> node:node -> parents:commit list -> task:task -> commit Lwt.t
       (** Create a new commit. *)
 
-      val node: t -> commit -> node Lwt.t
-      (** Get the commit node.
+      val node: t -> commit -> node option Lwt.t
+      (** [node t c] is [c]'s commit node or [None] is [c] is not
```

Comment:
"commit node" -> "root node"?
What if `c` is present in `t` but has no root node?
Do we need this function at all? The user can get the node from a commit easily enough using the commit API.